aspnetcore/fundamentals/environments.md

```diff
@@ -289,35 +248,12 @@ For Linux distros, use the `export` command at a command prompt for session-base
 
 Call <xref:Microsoft.Extensions.Hosting.HostingHostBuilderExtensions.UseEnvironment*> when building the host. See <xref:fundamentals/host/generic-host#environmentname>.
 
-
 ### Configuration by environment
 
-To load configuration by environment, we recommend:
-
-* *appsettings* files (*appsettings.{Environment}.json*). See <xref:fundamentals/configuration/index#json-configuration-provider>.
-* Environment variables (set on each system where the app is hosted). See <xref:fundamentals/host/generic-host#environmentname> and <xref:security/app-secrets#environment-variables>.
-* Secret Manager (in the Development environment only). See <xref:security/app-secrets>.
+To load configuration by environment, see <xref:fundamentals/configuration/index#json-configuration-provider>.
 
 ## Environment-based Startup class and methods
 
-### Inject IWebHostEnvironment into Startup.Configure
-
-Inject <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into `Startup.Configure`. This approach is useful when the app only requires adjusting `Startup.Configure` for a few environments with minimal code differences per environment.
-
-```csharp
-public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-{
-    if (env.IsDevelopment())
-    {
-        // Development environment code
-    }
-    else
-    {
-        // Code for all other environments
-    }
-}
-```
```

Comment:
I think it's still useful to show injecting `IWebHostEnvironment` into `Startup.Configure`. This is what the templates do and it's a bit easier if you only care about the environment when setting up the middleware pipeline.

Reply:
Added right after I show this in a snippet:

The preceding code:

* Injects <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into `Startup.Configure`. This approach is useful when the app only requires adjusting `Startup.Configure` for a few environments with minimal code differences per environment.
* Is similar to the code generated by the ASP.NET Core templates.